### ISSUE TITLE
A question about dipoloid genome assembly

### ISSUE THREAD

**Issue by ClarenceHsiang** [chhylp123/hifiasm#260]
Dear hifiasm developer teams:

Good afternoon.

In hifiasm output, we could get two haploid type assmebly, such as
XXX.asm.dip.hap1.p_ctg.gfa and XXX.asm.dip.hap2.p_ctg.gfa. 

Does it make sense that further polish and scaffolding, then merge these two assemblies as a diploid genome? 

Thank you so much.

Clarence

**Comment by chhylp123** [comment]
Yes, but probably no need to do polishing as current polishing tools are more likely to collapse repeats/different haplotypes.
